.travis.yml

```diff
@@ -1,12 +1,18 @@
 language: ruby
+
```

Comment:
Biggest nitpick ever: not sure it really needs the whitespace between each top level key

Reply:
i like a little breathing room 🎐